notes/05_val_reg/validation.tex

```diff
@@ -324,9 +324,9 @@ \subsubsection{Different views of the bias-variance tradeoff}
 \notesonly{
 \textbf{Remember: it's always a bias-variance \underline{tradeoff}}\\
 
-The above ``fixes'' rely on having concluded that the model is either overfitting or underfitting. And that this can potentially be mitigated by taking these different measures. However it is important to understand how overfitting and underfitting reflect where we are on the bias-variance tradeoff. This means that repeatedly applying one solution to an  overfitting model can eventually cause the model to underfit and vice-versa. One should also note that if one of the above doesn't work. It does not conclude that the model was not overfitting or underfitting. The above are simply guidelines on how to understand why a model has poor generalization performance and come up with simple techniques for finding a different bias-variance tradeoff.
```

Comment:
"However it is..." -> "However, it is..."

"applying one solution to an  overfitting model" -> "applying one solution to an overfitting model" (removed duplicates whitespace)